File: java-practice-prime/Loop_questions/Count_digit.java

public class Count_digit {
    public static int count(int num){
        int count=0;
        while(num>0){
            count++;
            num=num/10;
        }
        return count;
    }

    public static void main(String[] args) {
        int res=count(123654);
        System.out.println(res);
    }
    
}

File: 1.Interview_practce/string/countDigit.js
function countDigit() {
  let str = "my 2 name is 8 sudhir 4 kumar 6";
  let count = 0;
  for (let i = 0; i < str.length; i++) {
    if (str[i] >= "0" && str[i] <= "9") {
      count++;
    }
  }
  console.log(count);
}
countDigit();
